torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -3021,6 +3108,31 @@ def _finalize_params(fsdp_module: FullyShardedDataParallel) -> None:
                     # reset this flag for cases like "one forward pass + multiple backward passes"
                     self._post_backward_callback_queued = False
 
+        if (self.use_param_exec_order_policy() and self._param_exec_order_prep_stage):
+            self._param_exec_order_prep_stage = False
+            # Let the parameters in self._fsdp_params_exec_order ordered based on
+            # the execution order in the forward pass.
+            self._fsdp_params_exec_order.reverse()
+            for m in self.modules():
+                if m is not self and isinstance(m, FullyShardedDataParallel):
+                    if hasattr(m, "_use_param_exec_order_policy"):
+                        delattr(m, "_use_param_exec_order_policy")
+                    if hasattr(m, "_param_exec_order_prep_stage"):
+                        delattr(m, "_param_exec_order_prep_stage")
+            # TODO (linjianma): Construct a fsdp_wrap_map whose keys are all children modules with a FSDP wrap,
```

Comment:
nit: Generally its good to file + link to issues for these so that follow up tasks are clearly described and not forgotten.

Reply:
Just opened an issue regarding this: #79644